go.mod

```diff
@@ -35,18 +35,136 @@ require (
 	go.opentelemetry.io/otel/trace v1.2.0
 	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
 	golang.org/x/mod v0.5.1
-	k8s.io/api v0.22.2
-	k8s.io/apimachinery v0.22.2
-	k8s.io/client-go v0.22.2
-	k8s.io/component-base v0.22.2
-	k8s.io/klog/v2 v2.9.0
-	k8s.io/kubectl v0.22.2
+	k8s.io/api v0.23.0
+	k8s.io/apimachinery v0.23.0
+	k8s.io/client-go v0.23.0
+	k8s.io/component-base v0.23.0
+	k8s.io/klog/v2 v2.30.0
+	k8s.io/kubectl v0.23.0
 	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
-	sigs.k8s.io/cluster-api v1.0.2
-	sigs.k8s.io/cluster-api/test v1.0.2
-	sigs.k8s.io/controller-runtime v0.10.3
+	sigs.k8s.io/cluster-api v1.1.0-beta.1
+	sigs.k8s.io/cluster-api/test v1.1.0-beta.1
+	sigs.k8s.io/controller-runtime v0.11.0
 	sigs.k8s.io/kind v0.11.1
 	sigs.k8s.io/yaml v1.3.0
 )
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.0.2
+require (
```

Comment:
this might be a go 1.17 thing, the ones below are all indirect and were added when I ran go mod tidy

Reply:
yes: 
> At go 1.17 or higher:
The go.mod file includes an explicit require directive for each module that provides any package transitively imported by a package or test in the main module. (At go 1.16 and lower, an indirect dependency is included only if minimal version selection would otherwise select a different version.) This extra information enables module graph pruning and lazy module loading.
Because there may be many more // indirect dependencies than in previous go versions, indirect dependencies are recorded in a separate block within the go.mod file.

https://go.dev/ref/mod